apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/AIPolicyChat.tsx

```diff
@@ -175,7 +184,7 @@ export const AIPolicyChat = ({
             )}
           />
         </form>
-      </Form_Shadcn_>
+      </Form_Shadcn_> */}
```

Comment:
This is meant to be commented out right?

Reply:
oh yes! meant to remove it